recipe/meta.yaml

```
@@ -1,50 +1,52 @@
 {% set name = "prophet" %}
-{% set version = "1.1.5" %}
+{% set version = "1.2.1" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 80973c0b8a22d835bfa9d6665a78ebc63115135eaef0f73b46ee14e9bad3ca1a
+  sha256: 0c06a1de02396b95022aec60178d9348412c2799572b1b13315c6d045106471f
 
 build:
   number: 0
-  # LunarCalendar, which is a noarch package, requires ephem, unavailable on s390x
   # https://blog.quantinsti.com/installing-prophet-library-windows/
-  skip: true  # [py<37 or (linux and s390x)]
+  skip: true  # [py<37]
+  # cmdstan is not available for osx-arm64
+  skip: true  # [osx]
   script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
   binary_has_prefix_files:
     - {{ SP_DIR|replace("\\", "/") }}/stan_model/prophet_model.bin
   # To use our own cmdstan rather than download and build a different one:
   script_env:
     - PROPHET_REPACKAGE_CMDSTAN=false
-    - BOOST="${LIBRARY_INC}"  # [win]
+    - BOOST="${LIBRARY_INC}"        # [win]
 
 requirements:
   build:
-    - {{ compiler('cxx') }}  # [not win]
-    - {{ compiler('m2w64_cxx') }}  # [win]
-    - m2w64-make                   # [win]
-    - m2-coreutils                 # [win]
+    - {{ stdlib('c') }}             # [not win]
+    - {{ compiler('cxx') }}         # [not win]
+    - {{ compiler('ucrt64_cxx') }}  # [win]
+    - ucrt64-make                   # [win]
```

```suggestion
    - msys2-make                   # [win]
```

`msys2-` is always a better choice because it won't pollute the runtime with `gcc-foo` like `ucrt64-` does.